flake.nix

```diff
@@ -85,7 +87,7 @@
 
             (final: prev: {
               cargo-deluxe = cargo-deluxe.packages.${system}.default;
-              cargo-udeps = nixpkgs-unstable.legacyPackages.${system}.cargo-udeps;
```

Comment:
We don't need this one from nixos-unstable anymore, the stable one is enough for us.